packages/sdk-js/src/DidHelpers/common.ts

```diff
@@ -9,38 +9,25 @@
 // functions in this file are not meant to be public
 
 import type { ApiPromise } from '@polkadot/api'
-import type { SubmittableResultValue } from '@polkadot/api/types'
-import type { BlockNumber } from '@polkadot/types/interfaces'
-import { u8aToHex, u8aToU8a } from '@polkadot/util'
-
-import type {
-  FrameSystemEventRecord as EventRecord,
-  SpRuntimeDispatchError,
-} from '@kiltprotocol/augment-api'
+import type { SpRuntimeDispatchError } from '@kiltprotocol/augment-api'
 import { Blockchain } from '@kiltprotocol/chain-helpers'
-import {
-  resolver as DidResolver,
-  multibaseKeyToDidKey,
-  signersForDid,
-} from '@kiltprotocol/did'
-import type { Did, HexString } from '@kiltprotocol/types'
+import { multibaseKeyToDidKey } from '@kiltprotocol/did'
 
 import type {
   AcceptedPublicKeyEncodings,
   SharedArguments,
   TransactionHandlers,
-  TransactionResult,
 } from './interfaces.js'
 
-function mapError(err: SpRuntimeDispatchError, api: ApiPromise): Error {
+export function mapError(err: SpRuntimeDispatchError, api: ApiPromise): Error {
```

Comment:
this one and the next are only used in checkResult rn, so they could move there too